mkstage4.sh

```diff
@@ -129,66 +129,73 @@ shift;OPTIONS="$@"
 
 if [ ${S_KERNEL} -eq 1 ]
 then
-  USER_EXCL+=" --exclude=${TARGET}usr/src/* "
+  EXCLUDES_LIST+=" usr/src/*"
   if [ ${x86_64} -eq 1 ]
   then
-      USER_EXCL+=" --exclude=${TARGET}lib64/modules/* "
+      EXCLUDES_LIST+=" lib64/modules/*"
   else
-      USER_EXCL+=" --exclude=${TARGET}lib/modules/* "
+      EXCLUDES_LIST+=" lib/modules/*"
   fi
 fi
 
 
-# Excludes:
-EXCLUDES="\
---exclude=${TARGET}home/*/.bash_history \
---exclude=${TARGET}dev/* \
---exclude=${TARGET}var/tmp/* \
---exclude=${TARGET}media/* \
---exclude=${TARGET}mnt/*/* \
---exclude=${TARGET}proc/* \
---exclude=${TARGET}run/* \
---exclude=${TARGET}sys/* \
---exclude=${TARGET}tmp/* \
---exclude=${TARGET}var/lock/* \
---exclude=${TARGET}var/log/* \
---exclude=${TARGET}var/run/* \
---exclude=${TARGET}var/lib/docker/* "
+# Excludes - whitespace delimited list of things to leave out
+EXCLUDES_LIST="
+ home/*/.bash_history\
+ dev\
+ var/tmp\
+ media\
+ mnt\
+ proc\
+ run\
+ sys\
+ tmp\
+ var/lock\
+ var/log\
+ var/run\
+ var/lib/docker"
 
-EXCLUDES_DEFAULT_PORTAGE="\
- --exclude=${TARGET}var/db/repos/gentoo/* \
- --exclude=${TARGET}var/cache/distfiles/* \
- --exclude=${TARGET}usr/portage/*"
+EXCLUDES_DEFAULT_PORTAGE="
+ var/db/repos/gentoo/*\
+ usr/portage*\
+ var/cache/distfiles/*"
+
+# Excludes function - create tar --exclude=foo options
+exclude()
+{
+  ADDEXCLUDE=$(echo "$1" | sed 's/^\///')
+  EXCLUDES+=" --exclude=${TARGET}${ADDEXCLUDE}"
```

Comment:
This will break user excludes that don't intend to match the entire path below TARGET e.g. `-e '*.iso`